examples/fp8_sample/README.md

```diff
@@ -0,0 +1,96 @@
+### Usage demo:

```

Comment:
Suggest adding these examples to the weekly test or CI. cc @XuehaoSun @chensuyue